src/makeRobot.js

```diff
@@ -38,7 +38,69 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const Robot = {
+    name,
+    wheels,
+    version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      return (`name: ${this.name}, chip version: ${this.version},`
+      + ` wheels: ${this.wheels}`);
+    },
+    get location() {
+      return (`${this.name}: x=${this.coords.x}, y=${this.coords.y}`);
+    },
+    goForward(value = 1) {
+      if (value <= 0) {
+        this.coords.y += 0;
+      } else if (value > 0) {
+        this.coords.y += value;
```

Comment:
it's not necessary
use it in all cases
```suggestion
if (value > 0) {
        this.coords.y += value;
```